### PR TITLE
Dynamic hierarchical layout

### PR DESCRIPTION
**Pull request by CermakM**
### Highlights: 

feature | action
-- | --
pan & zoom | mouse and mousewheel events
focus by balancing to the center of mass | first mouse click
collapsible | second mouse click on the focused node
reset button | icon click in the top left corner

<br>

> Note: The state of the plots is not yet preserved on notebook reload or save

### CODE REVIEW

**Comment by goern** [comment]
holly molly what a large piece... I gonna check it out and test...

File: setup.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 from setuptools import find_packages
 from setuptools import setup
 
+_ASSETS_DIR = Path('assets/')
```
Comment (fridex) [inline setup.py]: This one is unused.

Comment (CermakM) [reply setup.py]: Thanks! Will reformat and rebase.

**Comment by CermakM** [comment]
Rebased, merging for follow up work to be continued in different PR as this seems to be big enough already.

File: setup.py [hunk 2]
```diff
@@ -37,11 +37,7 @@ def get_version():
...
         for p in find_packages('thoth/')
     ],
-    zip_safe=False,
+    include_package_data=True,
     install_requires=get_install_requires(),
-    command_options={
-        'build_sphinx': {
-            'version': ('setup.py', VERSION),
-        }
-    }
```
Comment (fridex) [inline setup.py]: This looks like a wrong rebase?

Comment (CermakM) [reply setup.py]: Oh my, I am so sorry, this is a terrible mistake, I must have deleted it while resolving a merge conflict. Thank you for spotting that.

Comment (fridex) [reply setup.py]: No problem at all. Thanks for the fix!

**Pull request merged**